### PR TITLE
Update the apt::key call to be compatible with puppetlabs/apt 2.x release

### PR DESCRIPTION
**Pull request by tampakrap**
Also:
- Use the released rspec-puppet Gem
- Use forge releases instead of latest master for the puppet modules fixtures

### CODE REVIEW

**Comment by tampakrap** [comment]
Another option for this would be just to remove the key_source, whatever you prefer :)

File: manifests/init.pp [hunk 1]
```diff
@@ -28,12 +28,14 @@
...
 
   apt::source { 'puppetlabs':
-    location   => 'http://apt.puppetlabs.com/',
-    key        => '1054B7A24BD6EC30',
-    key_source => 'https://apt.puppetlabs.com/pubkey.gpg',
-    pin        => '550',
-    repos      => $repo_list,
-    release    => $release,
+    location => 'http://apt.puppetlabs.com/',
+    key      => {
```
Comment (xaque208) [inline manifests/init.pp]: I suspect this is a change that should have a version dependency on the apt module.

File: .fixtures.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 fixtures:
-  repositories:
-    apt: "git://github.com/puppetlabs/puppetlabs-apt.git"
-    stdlib: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
+  forge_modules:
+    apt: "puppetlabs/apt"
```
Comment (xaque208) [inline .fixtures.yml]: While we're here, perhaps testing for the specific version we depend on.

Comment (tampakrap) [inline .fixtures.yml]: Mind pointing me to the documentation on how to do this? I'm unable to find it sorry

Comment (tampakrap) [inline .fixtures.yml]: I found it in the doc, but it is not possible to put there something like > 2.0.0

**Pull request closed without merging**